src/screens/WaitTimeScreen.tsx

```diff
@@ -1,42 +1,471 @@
+                <StyledText style={styles.poiLastUpdatedText}>
+                  {renderLastUpdated(poi.lastUpdated)}
+                </StyledText>
+              </Item>
+            ))}
+          </List>
+        </ScrollView>
+      </View>
+    </TouchableWithoutFeedback>
   );
 };
 
 const styles = StyleSheet.create({
-  huge: {
-    fontSize: 20,
+  searchContainer: {
+    paddingTop: 20,
+    backgroundColor: "#FFFFFF",
   },
-  container: {
-    flex: 1,
-    backgroundColor: "#fff",
+  searchBar: {
+    borderRadius: 6,
+    backgroundColor: "#F5F5F5",
+  },
+  search: {
+    backgroundColor: "#F5F5F5",
+  },
+  sortContainer: {
+    paddingTop: 20,
+  },
+  tags: {
+    paddingRight: 6,
+  },
+  sortByTag: {
+    borderRadius: 24,
+    width: "20%",
+    height: 34,
+    backgroundColor: "#FFFFFF",
+    marginRight: 6,
+  },
+  sortByText: {
+    fontSize: 15,
+    textAlign: "center",
+  },
+  modal: {
+    height: "100%",
+    width: "100%",
+    backgroundColor: "rgba(0,0,0,0)",
+  },
+  modalBackground: {
+    height: "100%",
+    width: "100%",
+    justifyContent: "flex-end",
+  },
+  modalButtons: {
+    color: "#1677FF",
+  },
+  modalHeader: {
+    width: "100%",
+    backgroundColor: "#ffffff",
+    borderTopLeftRadius: 10,
+    borderTopRightRadius: 10,
+    paddingBottom: 10,
+    paddingHorizontal: 20,
+    paddingTop: 20,
+  },
+  modalActionRow: {
+    flexDirection: "row",
+    justifyContent: "space-between",
+    paddingBottom: 20,
+  },
+  sortButtons: {
+    borderColor: "#FFFFFF",
+    alignItems: "flex-start",
+  },
+  checkIconTime: {
+    paddingLeft: "85%",
+  },
+  checkIconDistance: {
+    paddingLeft: "76%",
+  },
+  stepsContainer: {
+    marginTop: 20,
     alignItems: "center",
-    justifyContent: "center",
+    height: 20,
+  },
+  stepTextContainer: {
+    flexDirection: "row",
+    justifyContent: "space-between",
+  },
+  stepTextView: {
+    flex: 1,
+  },
+  stepTitleText: {
+    textAlign: "center",
+  },
+  stepDescriptionText: {
+    textAlign: "center",
+    color: "#999999",
+  },
+  stepScrollDivider: {
+    paddingTop: 30,
+    borderBottomColor: "#EEEEEE",
+    borderBottomWidth: 0.5,
+  },
+  poiWaitTimeText: {
+    textAlign: "right",
+    color: "#999999",
+  },
+  poiIcon: {
+    paddingRight: 15,
+    marginLeft: -10,
+  },
+  poiDistanceText: {
+    fontSize: 13,
+    color: "#999999",
+    paddingTop: 10,
+    paddingBottom: 5,
+  },
+  poiLastUpdatedText: {
+    fontSize: 13,
+    color: "#999999",
+    paddingTop: 5,
+    paddingBottom: 10,
   },
 });
 
+const StylesOverride = {
+  tagStyles: {
+    activeWrap: {
+      backgroundColor: "#1677FF",
+    },
+    activeText: {
+      color: "#F5F5F5",
+    },
+  },
+  stepsStyles: {
+    tail_last: {
+      width: 0,
+    },
+    head_default_s: {
+      width: 10,
+      height: 10,
+      backgroundColor: "#EEEEEE",
+      borderWidth: 0,
+    },
+    tail_blue: {
+      backgroundColor: "#EEEEEE",
```

Comment:
Anytime you're overriding the values here see if you can't change them in the theme provider values instead. I think the coloring for the tails _might_ be part of the theme. Could save you having to manually override them everywhere in the app.

Reply:
I think for tails and page-specific components I will leave them here because we don't want to overcrowd the theme file with just one-time values. I will move the ones we may reuse.